RECOGNITION.md

```diff
@@ -0,0 +1,21 @@
+# Recognizing contributions
+
+The most common form of individual contribution to the Flux project is
+time and effort. Thus, the project recognises people who reported
+issues, fixed bugs, wrote documentation, reviewed pull requests, and
+so on, by thanking them on the Flux website.
```

Comment:
```suggestion
so on, by thanking them [on the Flux website](https://fluxcd.io/#:~:text=Made%20possible%20by%20our%20contributors).
```

Reply:
These are contributors, not sponsors. Sponsors would e.g. have a dedicated page or other place where they would be highlighted.

Reply:
:+1: Yeah I could be more precise here

Reply:
Do we need to specify this in the doc?

I would suggest adding a sponsored by section on the homepage of fluxcd.io, We already have an adopters section now, featuring adopters logos. I think it's fair to add a similar section for sponsors too. Also probably easy since we have the code for adopters. Note that helm.sh has sponsor logos on the home page too.